tensorboard/webapp/widgets/histogram/histogram_linked_time_fob_test.ts

```diff
@@ -0,0 +1,176 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {NO_ERRORS_SCHEMA} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {By} from '@angular/platform-browser';
+import {ScaleLinear} from '../../third_party/d3';
+import {
+  AxisDirection,
+  Fob,
+  LinkedTimeFobControllerComponent,
+} from '../linked_time_fob/linked_time_fob_controller_component';
+import {LinkedTime} from '../linked_time_fob/linked_time_types';
+import {HistogramLinkedTimeFobController} from './histogram_linked_time_fob_controller';
+
+describe('HistogramLinkedTimeFobController', () => {
```

Comment:
Is there any coverage for histograms' getLowestStep() and getHighestStep()? I couldn't see any.